tracker.go

```diff
@@ -31,6 +31,7 @@ type mockProcessTracker struct {
 	Infos       []ProcessInfo
 }
 
+// nolint
```

Comment:
Why does this need nolint if it's used in linux tests?

Reply:
not sure